HLTrigger/Timer/plugins/ThroughputServiceClient.cc

```diff
@@ -3,7 +3,7 @@
 #include <cstring>
 
 // boost headers
-#include <boost/regex.hpp>
+#include <regex>
```

Comment:
Could you move this header to the "C++ headers" section, and remove the now empty "boost headers" section ?